README.md

```diff
@@ -19,16 +19,16 @@ Chrome has been shipping its own nonstandard segmentation API called `Intl.v8Bre
 let segmenter = new Intl.Segmenter("fr", {granularity: "word"});
 
 // Get an iterator over a string
-let iterator = segmenter.segment("Ceci n'est pas une pipe");
+let iterator = segmenter.break("Ceci n'est pas une pipe");
```

Comment:
I'm opposed to this rename, for two reasons.
1. As I noted in https://github.com/tc39/proposal-intl-segmenter/issues/59#issue-378577486 , I believe the term "break" associates more strongly with lines than with graphemes, words, or sentences (e.g., that the latter are separated not by "breaks" but by "boundaries").
2. For consistency with the rest of Intl, this method name should be a meaningful verb. But it's not a request to _break_ a string, it's a request to _segment_ one (cf. [UAX #29](https://www.unicode.org/reports/tr29/): "The process of boundary determination is also called segmentation").

Reply:
We decided we want to switch to iterating over breaks; I thought this name would serve that better than "segment".

Reply:
I understand, and am disagreeing with that choice of name. This method initiates the process of boundary determination, which the Unicode Standard defines as "segmentation". `segmenter.break(string)` reads as "use `segmenter` to _break_ `string`", whereas `segmenter.segment(string)` reads as "use `segmenter` to _segment_ `string`" (which is what it actually does).